src/components/accordion/accordion.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+    <div class="ui-accordion" role="heading" :class="rootClasses">
+        <button type="button"
+                class="u-reset ui-accordion--head"
+                v-if="!!$scopedSlots.head"
+                :aria-expanded="state.isOpen ? 'true' : 'false'"
+                @click.prevent="onRequestChange(!state.isOpen)">
+            <slot name="head" :is-open="state.isOpen"/>
+        </button>
+        <div class="ui-accordion--body"
+                role="presentation"
+                ref="body"
+                @transitionend="onTransitionEnd">
+            <slot name="body" :is-open="state.isOpen" :is-visible="isVisible"/>
+        </div>
+    </div>
+</template>
+
+<script>
+    import forceReflow from 'bianco.force-reflow';
+    import isMountedMixin from '../../mixins/is-mounted';
+    import bemMixin from '../../mixins/bem';
+
+    const TRANSITION_END_KEY_PROP = 'max-height';
+
+    export default {
+        mixins: [
+            isMountedMixin,
+            bemMixin('ui-accordion'),
+        ],
+        props: {
+            isOpen: {
+                type: Boolean,
+                default: false,
+            },
+            isPassive: {
+                type: Boolean,
+                default: false,
+            },
+        },
+        data() {
+            return {
+                state: {
+                    isAnimating: false,
+                    isOpen: this.isOpen,
+                },
+            };
+        },
+        computed: {
+            isVisible() {
+                return this.isClosing || this.hasFinishedOpeningAnimation;
+            },
+            hasFinishedOpeningAnimation() {
```

Comment:
maybe `hasOpened` is enough?

Reply:
I have removed the computed and replaced them with variables